### PR TITLE
Molecule Linear progress:

### PR DESCRIPTION
**Pull request by aodocs-backlight**
- Add a mode displaying only details (without the progress bar)

Molecule Tooltip:
- Add a simple tooltip component
- TODO: Handle positioning in the page regarding space left (top, left, right, bottom)

This component has 2 slots, one for the main display (the element that need to be hovered) and the tooltip itself

### CODE REVIEW

File: molecule_linear-progress/doc/index.html [hunk 1]
```diff
@@ -16,118 +16,87 @@
...
-    <style>
-      mwc-textfield {
-        min-width: 240px;
-      }
 
-      .shaped-filled {
-        --mdc-text-field-filled-border-radius: 32px 32px 0 0;
-      }
+<head>
+  <title>MWC Textfield Demo</title>
```
Comment (ratierd) [inline molecule_linear-progress/doc/index.html:21]: It should be molecule linear progress I think

File: molecule_linear-progress/doc/index.html [hunk 2]
```diff
@@ -16,118 +16,87 @@
...
+      opacity: 0;
+    }
+  </style>
+  <link href="../../shared/doc/shared.css" rel="stylesheet" />
+  <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" rel="stylesheet" />
+  <link href="https://fonts.googleapis.com/css?family=Material+Icons&display=block" rel="stylesheet" />
+  <style>
+    mwc-textfield {
+      min-width: 240px;
+    }
```
Comment (ratierd) [inline molecule_linear-progress/doc/index.html:41]: I'm not sure this is usefull is this context

File: molecule_linear-progress/doc/index.html [hunk 3]
```diff
@@ -16,118 +16,87 @@
...
+      --mdc-notched-outline-leading-border-radius: 28px 0 0 28px;
+      --mdc-notched-outline-trailing-border-radius: 0 28px 28px 0;
+    }
 
-      .demo-group-spaced {
-        flex-wrap: wrap;
-      }
+    mwc-textarea {
+      min-width: 250px;
+    }
```
Comment (ratierd) [inline molecule_linear-progress/doc/index.html:55]: Same, is this usefull ?

File: molecule_aodocs-tooltip/doc/index.html [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+    .flex {
+      display: flex;
+      justify-content: end;
+    }
+  </style>
+</head>
+
+<body class="unresolved">
+  <demo-header component="Linear Progress" package="linear-progress"></demo-header>
```
Comment (ratierd) [inline molecule_aodocs-tooltip/doc/index.html:57]: Tooltip here no ?

File: molecule_linear-progress/doc/index.html [hunk 5]
```diff
@@ -16,118 +16,87 @@
...
+
+    .demo-group-spaced>*,
+    [fullwidth] {
+      margin-top: 8px;
+    }
+  </style>
+</head>
+
+<body class="unresolved">
+  <demo-header component="Textfield" package="textfield"></demo-header>
```
Comment (ratierd) [inline molecule_linear-progress/doc/index.html:79]: Molecule linear progress

File: molecule_linear-progress/src/linear-progress.ts [hunk 6]
```diff
@@ -52,12 +55,17 @@ export class LinearProgress extends LitElement {
   protected override render(): TemplateResult {
     return html`
       ${this._displayTitle()}
-      <div class="flex" style="height: ${this.height}; width: ${this.width};">
-        ${this._renderParts()}
-      </div>
+      ${this._renderBar()}
```
Comment (ratierd) [inline molecule_linear-progress/src/linear-progress.ts:58]: Shouldn't this use mwc-linear-progress atom ? I'm confused

Comment (aodocs-backlight) [reply molecule_linear-progress/src/linear-progress.ts:58]: Nop, mwc-linear do not fit with our needs. We have to handle it manually. 

Indeed we need to have multiple values on the progress and mwc-linear progress allow us to only manage 2

**Pull request merged**